aeneas/src/arm/ArmRegSet.v3

```diff
@@ -65,7 +65,10 @@ component ArmMachRegs {
 		rnames[GPR] = "{gpr}";
 		rnames[ALL] = "{all}";
 
-		regs = MachRegSet.new(12, rarray, rnames, R12, CALLER_SPILL_START, CALLEE_SPILL_START);
+		var rclasses = Array<byte>.new(1);
```

Comment:
You should be able to just do this in one line with [GPR].

Reply:
Done